### PR TITLE
Add queryaction and cancelaction operations - closes #302

### LINKED ISSUE #302: How do you cancel or query the state of an action request?
In Mozilla's [Web Thing API](https://iot.mozilla.org/wot), an action can be requested using an HTTP `POST` request on an [Action resource](https://iot.mozilla.org/wot/#action-resource) to create an [ActionRequest resource](https://iot.mozilla.org/wot/#actionrequest-resource). The Action resource is essentially an action queue, consisting of multiple ActionRequest resources.

The response to the `POST` provides a unique URL for the ActionRequest resource, which can then have its status queried with a `GET` or be cancelled with a `DELETE`. A list of all current requests can be retrieved by a `GET` on the Action resource.

How would this API be described in a Thing Description following the current draft specification? Or is there another intended way to achieve these use cases?

### PR DESCRIPTION
**Pull request by benfrancis**
This is a WIP PR to add `queryaction` and `cancelaction` operations to the Thing Description, as discussed in #302.

Before landing this, we may want to decide the naming of potential corresponding meta-interactions at the top level forms level, such as `queryallactions`/`queryactionlist`/`queryallactionlists`. See https://github.com/w3c/wot-thing-description/issues/302#issuecomment-892836216 for further information.

***
<a href="https://pr-preview.s3.amazonaws.com/benfrancis/wot-thing-description/pull/1208.html" title="Last updated on Sep 14, 2021, 11:34 AM UTC (6cdcefb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1208/a2a435d...benfrancis:6cdcefb.html" title="Last updated on Sep 14, 2021, 11:34 AM UTC (6cdcefb)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6776,14 +6780,7 @@ <h1>JSON Schema for TD Instance Validation</h1>
...
                 "multipleOf": {
-                    "oneOf": [
-                        {
-                            "type": "integer"
-                        },
-                        {
-                            "type": "number"
-                        }
-                    ]
+                    "$ref": "#/definitions/multipleOfDefinition"
```
Comment (benfrancis) [inline index.html:6783]: Note that I didn't make this change, it was generated by the render script. Was this previously added in a source file without checking in the index.html output?

File: index.html [hunk 2]
```diff
@@ -6815,6 +6812,18 @@ <h1>JSON Schema for TD Instance Validation</h1>
...
+                {
+                    "type": "integer",
+                    "exclusiveMinimum": 0
+                },
+                {
+                    "type": "number",
+                    "exclusiveMinimum": 0
+                }
+            ]
+        },
```
Comment (benfrancis) [inline index.html:6826]: Same as above.

File: index.html (hunk 1 above)
Comment (JKRhb) [reply index.html:6783]: Sorry, this change has been recently introduced by #1204. Maybe there could be a separate regeneration on the `main` branch to differentiate it from your changes.

Comment (benfrancis) [reply index.html:6783]: No problem. I can either rebase on top of main once this is done, or even just land the changes in this PR now I know they are desired.

Comment (benfrancis) [reply index.html:6783]: (I've found it very confusing which files are meant to be edited directly vs. generated so it's easily done!)

**Comment by sebastiankb** [comment]
from today's TD call the group decide to merge

**Pull request merged**